velox/common/base/tests/SpillConfigTest.cpp

```diff
@@ -37,6 +37,7 @@ TEST(SpillConfig, spillLevel) {
       0,
       kInitialBitOffset,
       kNumPartitionsBits,
```

Comment:
Can you add tests for the new APIs? Thanks!

Reply:
Got it, added.